test/test04.erl

```diff
@@ -1,10 +1,11 @@
 -module(test04).
 
 -export([f/0, g/1, h/1]).
--export_type([tuple/2, int/0]).
+-export_type([tuple/2, int/0, int_tuple/0]).
```

Comment:
It is better to create a new file and testing in it. I think it is an easier way to resolve conflicts and simplify unit tests.

Reply:
I see!